buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -405,4 +425,16 @@ private static String destructiveDistroTestTaskName(ElasticsearchDistribution di
             distro.getFlavor(),
             distro.getBundledJdk());
     }
+
+    /**
+     * Determines whether Docker is available, for the purpose of defining packaging test tasks.
+     */
+    private static boolean isDockerAvailable() {
+        try {
+            return getDockerAvailability().isAvailable;
+        } catch (Exception e) {
```

Comment:
it's ok to fail in the case of some unexpected error. A failed build Today is better than realizing after 2 months that we were not testing on Docker when we should have